### PR TITLE
bpo-36722: Add What's New entry for debug ABI

### PR DESCRIPTION
**Pull request by vstinner**
https://bugs.python.org/issue36722

### CODE REVIEW

**Comment by vstinner** [comment]
@elprans @pganssle: Would you mind to review this change? I'm not sure about the length of the text. I chose to document the 3 changes in 3 paragraph since each change is very subtle and it might help readers to have a link to the bpo issue.

File: Doc/whatsnew/3.8.rst [hunk 1]
```diff
@@ -98,6 +98,30 @@ subdirectories).
 
 (Contributed by Carl Meyer in :issue:`33499`.)
 
+Debug build uses the same ABI than release build
```
Comment (auscompgeek) [inline Doc/whatsnew/3.8.rst]: s/than/as/?

Comment (vstinner) [reply Doc/whatsnew/3.8.rst]: Right, fixed.

**Comment by vstinner** [comment]
Oh. I didn't know that there is a new Python 3.8 release (alpha4) next monday. I prefer to merge this What's New In Python 3.8 soon and rework the text later, than not document this change. So I merged my change.

**Pull request merged**